app/_src/gateway-operator/get-started/konnect/deploy-data-plane.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Deploy a Data Plane
+content-type: tutorial
+book: kgo-konnect-get-started
+chapter: 2
+---
+
+In order for {{ site.kgo_product_name }} data planes to attach to {{ site.konnect_short_name }} they need to know which endpoint to connect to, and how to authenticate their requests.
+
+To get this information, [log in to {{ site.konnect_short_name }}](https://cloud.konghq.com/login) and visit _Gateway Manager_. Select a Control Plane and then create a _New Runtime Instance_.
+
+Follow the instructions on the screen, but stop once you have created a secret in step 3:
+
+```bash
+kubectl create secret tls kong-cluster-cert -n kong --cert=/{PATH_TO_FILE}/tls.crt --key=/{PATH_TO_FILE}/tls.key
+```
+
+The instructions in {{ site.konnect_short_name }} use Helm to deploy a dataplane instance, but we want to use {{ site.kgo_product_name }}.
```

Comment:
Delete this sentence

Reply:
Why should it be deleted? I think it's useful to call out the KGO is a replacement for Helm